src/main/java/com/socialfeed/domain/DataProvider.java

```diff
@@ -71,35 +87,123 @@
 		HashSet<String> ids = new HashSet<String>();
 		try
 		{
-			if (tableName.contains("Connection"))
+			String urlName = EndpointConstants.HOST + ":";
+			Gson g = new Gson();
+			//List<BaseDatabaseObject> relationships = null;
+			URL url = null;
+			HttpURLConnection conn = null;
+			String data;
+			ObjectMapper objectMapper = new ObjectMapper();
+			objectMapper.registerModule(new JodaModule());
+			TypeFactory typeFactory = objectMapper.getTypeFactory();	
+			switch(tableName)
 			{
-				resultSet = sqlConnection.getTableRowByColumnName(tableName, "user1", id);
-				DataProvider.updateIdSet(resultSet, ids);
+			case DataProvider.CONNECTION:
+				String getUserIds = urlName + EndpointConstants.CORE_PORT  + EndpointConstants.USER_CONNECTION;
+				url = new URL(getUserIds + String.format("?user1=%s&type=Friend", id));
+				conn = (HttpURLConnection) url.openConnection();
+				
+				data = DataProvider.getJsonDataFromConnection(conn);
+				conn.disconnect();
+				System.out.println(data);
+				//ArrayList<UserConnection> userConnections = g.fromJson(data, new TypeToken<ArrayList<UserConnection>>(){}.getType());
+
+				ArrayList<UserConnection> userConnections = objectMapper.readValue(data,  typeFactory.constructCollectionType(ArrayList.class, UserConnection.class));
+				for (UserConnection userConnection: userConnections)
+				{
+					System.out.println("User Id " + userConnection.getId());
+					System.out.println(String.format("Adding User2 %s", userConnection.getUser2()));
+					ids.add(userConnection.getUser2());
+				}
+				
+//				for (UserConnection userConnection: jacksonUserConnections)
+//				{
+//					System.out.println(String.format("Jackson Adding User2 %s", userConnection.getUser2()));
+//					//ids.add(userConnection.getUser2());
+//				}
 				
-				resultSet = sqlConnection.getTableRowByColumnName(tableName, "user2", id);
-				DataProvider.updateIdSet(resultSet, ids);
+				url = new URL(getUserIds + String.format("?user2=%s&type=Friend", id));
+				conn = (HttpURLConnection) url.openConnection();
+				data = DataProvider.getJsonDataFromConnection(conn);
+				//userConnections = g.fromJson(data, new TypeToken<List<UserConnection>>(){}.getType());
+				userConnections = objectMapper.readValue(data,  typeFactory.constructCollectionType(ArrayList.class, UserConnection.class));
+				for (UserConnection userConnection: userConnections)
+				{
+					System.out.println("User 2 Id " + userConnection.getId());
```

Comment:
will have to convert these to logs if they are important

Reply:
Yeah we do.